modules/oadp-secrets-for-different-credentials.adoc

```diff
@@ -0,0 +1,234 @@
+
+. Create a `credentials-velero` file for the snapshot location in the appropriate format for your cloud provider.
+. Create a `Secret` for the snapshot location with the default name:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc create secret generic {credentials} -n openshift-adp --from-file cloud=credentials-velero
+----
+
+. Create a `credentials-velero` file for the backup location in the appropriate format for your object storage.
+. Create a `Secret` for the backup location with a custom name:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc create secret generic <custom_secret> -n openshift-adp --from-file cloud=credentials-velero
+----
+
+. Add the `Secret` with the custom name to the `DataProtectionApplication` CR, as in the following example:
+endif::[]
+ifdef::installing-oadp-azure[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa-sample>
+  namespace: openshift-adp
+spec:
+...
+  backupLocations:
+    - velero:
+        configuration:
+          resourceGroup: <azure_resource_group>
+          storageAccount: <azure_storage_account_id>
+          subscriptionId: <azure_subscription_id>
+          storageAccountKeyEnvVar: AZURE_STORAGE_ACCOUNT_ACCESS_KEY
+        credential:
+          key: cloud
+          name: <custom_secret> <1>
+        provider: azure
+        default: true
+      objectStorage:
+        bucket: <bucket_name>
+        prefix: <prefix>
+  snapshotLocations:
+    - velero:
+        config:
+          resourceGroup: <azure_resource_group>
+          subscriptionId: <azure_subscription_id>
+          incremental: true
+        name: default
+        provider: {provider}
+----
+<1> Backup location `Secret` with custom name.
+endif::[]
+ifdef::installing-oadp-gcp[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa-sample>
+  namespace: openshift-adp
+spec:
+...
+  backupLocations:
+    - velero:
+        provider: {provider}
+        default: true
+        credential:
+          key: cloud
+          name: <custom_secret> <1>
+        objectStorage:
+          bucket: <bucket_name>
+          prefix: <prefix>
+  snapshotLocations:
+    - velero:
+        provider: {provider}
+        default: true
+        config:
+          project: <project>
+          snapshotLocation: us-west1
+----
+<1> Backup location `Secret` with custom name.
+endif::[]
+ifdef::installing-oadp-mcg[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa-sample>
+  namespace: openshift-adp
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+    restic:
+      enable: true
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: minio
+          s3Url: <https://www.example.com>
```

Comment:
```suggestion
          s3Url: <url>
```
You could say more to describe the URL (`<descriptive_word_url>`), but should be formatted like any other user replaceable text.